package.json

```diff
@@ -10,7 +10,7 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "ipad": "react-native run-ios --simulator=\"iPad Pro (12.9-inch) (4th generation)\"",
-    "desktop": "webpack --config config/webpack/webpack.dev.js --platform desktop && electron desktop/main.js",
+    "desktop": "concurrently --kill-others-on-fail -n Renderer,Main \"webpack-dev-server --config config/webpack/webpack.dev.js --port 8080 --platform desktop\" \"wait-port localhost:8080 && export NODE_ENV=development && electron desktop/main.js\" ",
```

Comment:
Why do you need to specify port 8080? This won't work if you're already running the web app on port 8080.

Reply:
Actually, I stand corrected. It seems to work just fine

Reply:
We need to pass a port here https://github.com/Expensify/Expensify.cash/blob/278576c5b44b65575ac6d1f24ae4163477e01ed4/desktop/main.js#L56.
So it's better if we explicitly specify a port here otherwise user will not have any idea if webpack-dev-server starts the server on some other port. He will see a blank electron window.

Reply:
It's actually better this way I think. If you run `npm run web` in two terminal tabs, it will run the app twice – once on port 8080 and once on port 8081. This way, it just runs once and the desktop client uses the same port that the web client is using.